implementation/ProbDistLeftDomainBoundary.hpp.in

```diff
@@ -0,0 +1,64 @@
+#ifndef MATH_IMPLEMENTATION_PROBABILITYDISTRIBUTIONLEFTDOMAINBOUNDARY_HPP
+#define MATH_IMPLEMENTATION_PROBABILITYDISTRIBUTIONLEFTDOMAINBOUNDARY_HPP
+
+#include "math/API/ProbabilityDistribution.hpp"
+#include "math/implementation.hpp"
+
+namespace math {
+namespace implementation {
+
```

Comment:
I only know what this does because I saw the pull request comments. Other users won't have that luxury. This needs to be documented including how it should be used.